docker/build/Dockerfile

```diff
@@ -23,6 +23,12 @@ RUN \
 
 WORKDIR /
 
+# Install garble for obfuscated builds
+RUN \
```

Comment:
Thanks for this PR! Could you move this go install to `toolchain/bootstrap_pure.sh`, probably next to the line: `echo "Bootstrapping is no longer needed for go 1.20+"` is best. As garble states that go1.20 is required, this means that garble only is installed for go1.20+ You may also wish to add some conditional in `xgo.go` so that the flags are only accepted for the specific go versions.

Reply:
Good call, I can do that